packages/model/src/research-output.ts

```diff
@@ -1,7 +1,15 @@
 import { TeamResponse } from './team';
 import { ListResponse } from './common';
 
-export type ResearchOutputType = 'proposal';
+export type ResearchOutputType =
+  | 'proposal'
```

Comment:
@fampinheiro You said this should match the values shown in squidex which i assume to have proper casing

Reply:
On squidex we have proper case :). Can you change it to Proposal?